nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -1428,16 +1428,30 @@ def __init__(
             self._logger = logger
 
     def cc_workflow(self, output_path):
+        print("Writing deduped parsed id")
+        st = time.time()
         deduped_parsed_id_path = self._write_dedup_parsed_id()
+        print(f"Time taken for writing deduped parsed id = {time.time()-st}s")
```

Comment:
We don't need to duplicate these since `_write_dedup_parsed_id` already has the logger. See line 1633